swarm/src/lib.rs

```diff
@@ -68,6 +68,7 @@ pub mod dial_opts;
 pub mod dummy;
 pub mod handler;
 mod listen_opts;
+pub mod translation;
```

Comment:
```suggestion
mod translation;
```
Lets make this private if we are re-exporting anyway.